docs-vim.js

```diff
@@ -1,6 +1,7 @@
 vim = {
     "mode": "insert",
     "num": "",
+    "currentSequence": "",
     "keys": {
```

Comment:
Nitpick: Not completely related to this PR, but I think this could benefit from a few short comments describing each of the elements in this dictionary/object. At least `num`, on a first read it's not clear what we'll be using it for. Maybe something along the lines of "User-entered number buffer for the number of times to repeat a key press"? (I could be misunderstanding its use)